--- core/search_parser_test.go
+++ core/search_parser_test.go
@@ -12,14 +12,14 @@
	results, errors := ParseSearch(reader)

	if len(errors) > 1 {
-		t.Errorf("Expected 1 errors but got %d\n", len(errors))
+		t.Errorf("Expected 0 errors but got %d\n", len(errors))
		for _, parseError := range errors {
			t.Log(parseError)
		}
	}

-	if len(results) != 57 {
-		t.Errorf("Expected 57 results but got %d\n", len(results))
+	if len(results) != 59 {
+		t.Errorf("Expected 59 results but got %d\n", len(results))
	}
 }

@@ -28,14 +28,14 @@
	results, errors := ParseSearchV2(reader)

-	if len(errors) != 1 {
-		t.Errorf("Expected 1 errors but got %d\n", len(errors))
+	if len(errors) != 0 {
+		t.Errorf("Expected 0 errors but got %d\n", len(errors))
		for _, parseError := range errors {
			t.Log(parseError)
		}
	}

-	if len(results) != 57 {
-		t.Errorf("Expected 57 results but got %d\n", len(results))
+	if len(results) != 59 {
+		t.Errorf("Expected 59 results but got %d\n", len(results))
	}
 }
